docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/amazon.md

```diff
@@ -162,3 +166,129 @@ Setting the value of the tag to `owned` tells the cluster that all resources wit
+
+### Using Out of Tree AWS Cloud Provider for RKE1
+
+1. [Node name conventions and other prerequisities ](https://cloud-provider-aws.sigs.k8s.io/prerequisites/) must be followed for cloud provider to find the instance correctly. Rancher provisioned clusters don't support configuring `providerID`. 
+
+> When IP-based naming is used, the nodes must be named after the instance followed by the regional domain name (`ip-xxx-xxx-xxx-xxx.ec2.<region>.internal`). If you have custom domain name set in the DHCP options, you must set `--hostname-override` on `kube-proxy` and `kubelet` to match the above-mentioned naming convention. 
+
+When creating a [Custom cluster](../../../../pages-for-subheaders/use-existing-nodes.md), you have to add  [`--node-name`](../../reference-guides/cluster-configuration/rancher-server-configuration/use-existing-nodes/rancher-agent-options) in the `docker run` node registration command to set `hostname-override` (For e.g, `"$(hostname -f)"`). This can be done manually or by using `Show Advanced Options` in Rancher UI to add `Node Name`. 
+
+2. Select `External (out-of-tree)` cloud provider, which sets `--cloud-provider=external` for Kubernetes components. This can also be done manually by setting it in `cluster.yml`.
+
+```
+rancher_kubernetes_engine_config:
+  cloud_provider:
+    name: external
+```
+
+3. Install AWS cloud controller manager after cluster provisions. Note that cluster isn't successfully provisioned and nodes are tainted `uninitialized` until cloud controller manager is deployed.  This can be done via Helm charts in UI or manually.
+
+### Helm chart install
+1. Add repository `https://kubernetes.github.io/cloud-provider-aws` under Apps -> Repositories. 
+
+2. Install `aws-cloud-controller-manager ` under Apps -> Charts. 
+
+3. Select namespace `kube-system` and enable `Customize Helm options before install`. 
+
+4. Rancher provisioned RKE nodes are tainted `node-role.kubernetes.io/controlplane` so you have to update tolerations and nodeSelector.
+
+```
+tolerations:
+  - effect: NoSchedule
+    key: node.cloudprovider.kubernetes.io/uninitialized
+    value: 'true'
+  - effect: NoSchedule
+    value: 'true'
+    key: node-role.kubernetes.io/controlplane
+
+```
+NOTE: There's currently [a known issue](https://github.com/rancher/dashboard/issues/9249) in UI which doesn't allow nodeSelector to be updated, proceed with install chart and then edit daemonset manually to set the `nodeSelector`.  
+
+```
+nodeSelector:
+  node-role.kubernetes.io/controlplane: 'true'
+```
+
+5. Add `get` to `clusterRoleRules` for cloud controller manager to get service accounts on startup. 
+
+```
+  - apiGroups:
+      - ''
+    resources:
+      - serviceaccounts
+    verbs:
+      - create
+      - get
+```
+
+6. Update container args 
+```
+  - '--use-service-account-credentials=true'
+  - '--configure-cloud-routes=false'
+```   
+
+7. Install chart and confirm daemonset `aws-cloud-controller-manager` deploys successfully.
```

Comment:
This needs a command + output to make sure the reader knows it was deployed successfully to avoid issues later (thinking it was deployed successfully while it was not)